MAUI/NavigationDrawer/Side-Pane-Content.md

```diff
@@ -0,0 +1,233 @@
+
+*	Footer Content
+
+N> Header and Footer content are optional but Drawer content is mandatory to allocate space for the drawer.
+		
+## Header Content
+
+As the name suggests it is displayed at the top of drawer. The `DrawerHeaderView` property is used to set the header content of drawer. 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<navigationdrawer:SfNavigationDrawer x:Name="navigationDrawer">
+    <navigationdrawer:SfNavigationDrawer.DrawerSettings>
+        <navigationdrawer:DrawerSettings>
+            <navigationdrawer:DrawerSettings.DrawerHeaderView>
+                <Grid BackgroundColor="#1aa1d6">
+                    <VerticalStackLayout VerticalOptions="Center"
+                                         HorizontalOptions="Center">
+                        <Label Text="Header View"/>
+                    </VerticalStackLayout>
+                </Grid>
+            </navigationdrawer:DrawerSettings.DrawerHeaderView>
+        </navigationdrawer:DrawerSettings>
+    </navigationdrawer:SfNavigationDrawer.DrawerSettings>
+</navigationdrawer:SfNavigationDrawer>
+
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfNavigationDrawer navigationDrawer = new SfNavigationDrawer();
+DrawerSettings drawerSettings = new DrawerSettings()
+{
+    DrawerWidth = 250,
+};
+navigationDrawer.DrawerSettings = drawerSettings;
+this.Content = navigationDrawer;
+  
+{% endhighlight %}
+
+{% endtabs %}
+
+![Header]()
+
+## Header Height
+
+The height of the drawer header content can be adjusted using the `DrawerHeaderHeight` property.
+
+N> `DrawerHeaderView` will be disabled by setting the `DrawerHeaderHeight` to zero.
+
+{% tabs %}
+
+{% highlight xaml %}
+    
+<navigationdrawer:SfNavigationDrawer x:Name="navigationDrawer">
+    <navigationdrawer:SfNavigationDrawer.DrawerSettings>
+        <navigationdrawer:DrawerSettings DrawerHeaderHeight="150">
+            <navigationdrawer:DrawerSettings.DrawerHeaderView>
+                <Grid BackgroundColor="#1aa1d6">
+                    <VerticalStackLayout VerticalOptions="Center"
+                                         HorizontalOptions="Center">
+                        <Label Text="Header View"/>
+                    </VerticalStackLayout>
+                </Grid>
+            </navigationdrawer:DrawerSettings.DrawerHeaderView>
+        </navigationdrawer:DrawerSettings>
+    </navigationdrawer:SfNavigationDrawer.DrawerSettings>
+</navigationdrawer:SfNavigationDrawer>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfNavigationDrawer navigationDrawer = new SfNavigationDrawer();
+DrawerSettings drawerSettings = new DrawerSettings()
+{
+    DrawerHeaderHeight = 150,
+    DrawerWidth = 250,
+};
+navigationDrawer.DrawerSettings = drawerSettings;
+this.Content = navigationDrawer;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Header height]()
+
+## Footer Content
+
+As the name suggests it is displayed at the bottom of the drawer. The `DrawerFooterView` property is used to set the footer content of the drawer. 
```

Comment:
As the name suggests, it is displayed at the bottom of the drawer. The `DrawerFooterView` property is used to set the footer content of the drawer.

Reply:
Changed